apply_pose: True
preprocess: True
min_range: 2.0
max_range: 10.0

pcl_topic: "/camera/rgb/points"

# TF2
parent_frame: "world"
child_frame: "kinect"
